### PR TITLE
Small changes in layout, fixed couple of issues, changed some features

### PR DESCRIPTION
**Pull request by manish0338**
1.  Changed layout 
2. added spaces for better alignment
3. changed balance format
4. removed dependency from wallet.sh
5. used rpc package from go-sdk for rpc calls
6. changed error handing for all the parts (now TUI runs if rpc call fail, logs are not in the place, local node is down)

### CODE REVIEW

File: src/data/dataStore.go [hunk 1]
```diff
@@ -3,9 +3,11 @@ package data
 import(
 	"strconv"
 	"time"
+	"encoding/json"
 	
-	"github.com/harmony-one/harmony-tui/src/rpc"
+	"github.com/harmony-one/go-sdk/pkg/rpc"
 	"github.com/harmony-one/harmony-tui/config"
+	//"github.com/harmony-one/go-sdk/pkg/sharding"
```
Comment (LeoHChen) [inline src/data/dataStore.go]: remove comment

File: src/data/dataStore.go [hunk 2]
```diff
@@ -44,7 +49,7 @@ func refreshData(){
 		case <- ticker.C:
 			latestHeader, err := rpc.Request("hmy_latestHeader", config.HmyURL, []interface{}{})
 			if err!=nil {
-				Quitter(err.Error() + " to hmy_latestHeader")
+				//Quitter(err.Error() + " to hmy_latestHeader")
```
Comment (LeoHChen) [inline src/data/dataStore.go]: remove comment

File: widgets/instanceInfo.go [hunk 3]
```diff
@@ -17,34 +17,33 @@ import (
...
+	
 	
 	wrapped, err := text.New(text.WrapAtRunes())
 	if err != nil {
 		panic(err)
 	}
 
-	if err := wrapped.Write(appVersion + "", text.WriteCellOpts(cell.FgColor(cell.ColorGreen))); err != nil {
-		panic(err)
+	data.AppVersion, err = src.Exec_cmd(config.HarmonyPath + "./harmony -version")
```
Comment (LeoHChen) [inline widgets/instanceInfo.go:26]: will it work without LD_LIBRARY_PATH=. env setting?

**Review by LeoHChen — COMMENTED** [review]
please regenerate go.mod using 'go mod tidy'

also please fix the code format using 'goimports' tool.

File: src/data/dataStore.go (hunk 2 above)
Comment (manish0338) [reply src/data/dataStore.go]: removed

File: src/data/dataStore.go (hunk 1 above)
Comment (manish0338) [reply src/data/dataStore.go]: removed

File: widgets/instanceInfo.go (hunk 3 above)
Comment (manish0338) [reply widgets/instanceInfo.go:26]: yes, src/shell.go/Exec_cmd function is handling it

**Review by LeoHChen — APPROVED** [review]
I am approving now. but the binary has to be removed from the repo.

**Pull request merged**